content/applications/~larryx-woldyr/osmosis.md

```diff
@@ -0,0 +1,12 @@
++++
+title = "Osmosis"
+shortcode = "~larryx-woldyr/osmosis"
+license = "MIT"
+image = "https://holium-app-globs.nyc3.cdn.digitaloceanspaces.com/osmosis/osmosis-logo-wc.png"
+bgColor = "#180D39"
+developer = "~larryx-woldyr"
+website = "https://https://osmosis.zone/"
+description = "Interchain liquidity lab."
++++
+
+A fully self-hosted Osmosis frontend, deployed via Urbit.
```

Comment:
Best way to reference the grant is to just add a line here with a link to it. There's no official metadata to reference this to my knowledge.

Reply:
I saw it via this grant page: https://urbit.org/grants/urbitswap